.github/workflows/all.yml

```diff
@@ -171,6 +171,11 @@ jobs:
               && go install)
           sudo apt-get update -y
           sudo apt-get install -y fakeroot rpm
+      - name: Generate Windows binary metadata
+        run: |
+          (cd && go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo)
```

Comment:
```suggestion
          (cd && go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo@1.2.0)
```
we probably should pin the version we install, just in case...

Reply:
Resolved in latest push.